samples/snippets/src/main/java/com/example/spanner/GetCommitStatsSample.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+// [START spanner_get_commit_stats]
+import com.google.cloud.spanner.CommitResponse;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.Options;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import java.util.Arrays;
+import java.util.concurrent.ExecutionException;
+
+public class GetCommitStatsSample {
+
+  static void getCommitStats() throws InterruptedException, ExecutionException {
+    // TODO(developer): Replace these variables before running the sample.
+    final String projectId = "my-project";
+    final String instanceId = "my-instance";
+    final String databaseId = "my-database";
+    final String table = "Albums";
+
+    try (final Spanner spanner =
+        SpannerOptions.newBuilder().setProjectId(projectId).build().getService()) {
+      final DatabaseClient databaseClient = spanner
+          .getDatabaseClient(DatabaseId.of(projectId, instanceId, databaseId));
+      getCommitStats(databaseClient, table);
+    }
+  }
+
+  static void getCommitStats(DatabaseClient databaseClient, String table) {
```

Comment:
nit: I think it would be better to not have `table` as an input parameter, but to set it in the method itself. See for example https://cloud.google.com/spanner/docs/samples/spanner-update-data#spanner_update_data-java